python/packagescanner_test.go

```diff
@@ -70,12 +70,6 @@ var scanTable = []test.ScannerTestcase{
 		Hash:    "sha256:109a55eba749c02eb6a4533eba12d8aa02a68417ff96886d049798ed5653a156",
 		Scanner: &python.Scanner{},
 	},
-	{
```

Comment:
how come this was dropped?

Reply:
Added back, good catch. It didn't exist when I went back to browse the old fixtures